### PR TITLE
i18n(tr): add `customization.mdx`

### PR DESCRIPTION
**Pull request by erkanaltinors**
**What kind of changes does this PR include?**

Updates of Starlight docs site content

**Description**

This pull request updates translated in Turkish `customization.mdx` file in project.

### CODE REVIEW

File: docs/src/content/docs/tr/guides/customization.mdx [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+  </TabItem>
+  <TabItem label="Genel yapılandırma">
+
+```js {7}
+// astro.config.mjs
+
+defineConfig({
+	integrations: [
+		starlight({
+			title: 'Docs with custom table of contents config',
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/guides/customization.mdx:154]: This title may have been skipped as all other translated?

File: docs/src/content/docs/tr/guides/customization.mdx [hunk 2]
```diff
@@ -0,0 +1,464 @@
...
+2. `src/fonts/font-face.css` içindeki her bir yazı karakteri için [`@font-face` deklarasyonu](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face) ekleyin.
+    Yazı karakteri dosyasına ait dosya yolunu `url()` fonksiyonu içinde kullanın.
+
+
+   ```css
+   /* src/fonts/font-face.css */
+
+   @font-face {
+   	font-family: 'Custom Font';
+   	/* Use a relative path to the local font file in `url()`. */
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/guides/customization.mdx:344]: I guess we could translate this comment too?

File: docs/src/content/docs/tr/guides/customization.mdx [hunk 3]
```diff
@@ -0,0 +1,464 @@
...
+
+#### Fontsource yazı karakteri ayarlama
+
+[Fontsource](https://fontsource.org/) projesi, Google Fonts ve diğer açık kaynaklı yazı karakterlerini kullanmayı basitleştirir.
+Kullanmak istediğiniz yazı karakterlerini ve projenize ekleyeceğiniz okumaya hazır CSS dosyalarını içeren npm modülü olarak sunar.
+
+<Steps>
+
+1.  [Fontsource’un kataloğundan](https://fontsource.org/) kullanmak istediğiniz yazı karakterini bulun.
+    Bu örnek IBM Plex Serif](https://fontsource.org/fonts/ibm-plex-serif) kullanır.
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/guides/customization.mdx:382]:
```suggestion
    Bu örnek [IBM Plex Serif](https://fontsource.org/fonts/ibm-plex-serif) kullanır.
```

Small link formatting issue

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for another translation :thumbsup:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view) and except for the few minor suggestions, the translation looks good to me.

**Comment by erkanaltinors** [comment]
All suggested changes applied @HiDeoo 🚀

File: docs/src/content/docs/tr/guides/customization.mdx [hunk 4]
```diff
@@ -0,0 +1,464 @@
...
+<Steps>
+
+1. `src/fonts/` dizininde boş `font-face.css` dosyası oluşturun ve yazı karakteri dosyalarınızı ekleyin:
+
+   <FileTree>
+
+   - src/
+     - content/
+     - fonts/
+       - **CustomFont.woff2**
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/guides/customization.mdx:329]:
```suggestion
       - **OzelYaziKarakteri.woff2**
```

One last change to match your last commit translation, we need to update it in the file tree.

Comment (erkanaltinors) [reply docs/src/content/docs/tr/guides/customization.mdx]: Applied.

**Review by HiDeoo — APPROVED** [review]
Thanks for the updates, looks perfect now, let's ship it :rocket:

**Pull request merged**